app/controllers/reviews_controller.rb

```diff
@@ -16,23 +16,26 @@ def create
 
   def edit
     @podcast = Podcast.find(params[:podcast_id])
-    @review = Review.find(params[:id])
-    @new_review = @review
+    @edit_review = Review.find(params[:id])
+    @new_review = @edit_review
     @reviews = @podcast.reviews
     render :'/podcasts/show'
   end
 
   def update
-    @review = Review.find(params[:id])
-    @new_review = Review.new(review_params)
-    @new_review.podcast = @podcast
-    @new_review.user = current_user
+    @edit_review = Review.find(params[:id])
     @podcast = Podcast.find(params[:podcast_id])
-    if @review.update(review_params)
+    if @edit_review.update(review_params)
       flash[:notice] = "Review updated successfully"
+      @new_review = Review.new(review_params)
+      @new_review.podcast = @podcast
+      @new_review.user = current_user
       redirect_to podcast_path(@podcast)
     else
-      flash.now[:notice] = "Review not updated"
+      flash.now[:notice] = @edit_review.errors.full_messages.to_sentence
+      @podcast = Podcast.find(params[:podcast_id])
```

Comment:
Your `update` path, remember, doesn't actually _show_ anything. It looks like some of this code (for the ordered `@reviews`, for instance) belongs in your `show` action, which is the view where your reviews are actually displayed. Please consider & try to clean this up in a future PR!